spec/lib/traject/config/folio_course_reserves_spec.rb

```diff
@@ -34,4 +34,21 @@
       'EDUC-147-01 -|- Stanford and Its Worlds: 1885-present -|- Mitchell Stevens'
     ]
   end
+
+  context 'for item display' do
+    # Filter to the item that's on reserve so we can check its item_display
+    subject(:item_display) { result['item_display_struct'].find { |item| item['barcode'] == '36105232609540' } }
+
+    it 'adds the course ID' do
+      expect(item_display['course_id']).to eq 'HISTORY-58E-01'
```

Comment:
Could this all be one test with
```ruby
expect(item_display).to include('course_id' => 'HISTORY-58E-01',
                                'reserve_desk' => 'GRE-CRES',
                                'loan_period' => '2-hour loan')
```

Reply:
Sure, but I like having the `it` declarations be more atomic. More granular tests is better to me.

Reply:
OK, I can do that.

Reply:
Done